management/src/main/java/org/kaazing/gateway/management/config/ServiceDefaultsConfigurationBeanImpl.java

```diff
@@ -95,35 +99,120 @@ public String getAcceptOptions() {
                     jsonOptions.put("ssl.verify-client", "none");
                 }
 
-                jsonOptions.put("ws.maximum.message.size", acceptOptions.get("ws.maximum.message.size"));
+                jsonOptions.put("ws.maximum.message.size", acceptOptions.remove("ws.maxMessageSize"));
```

Comment:
ws.maximum.message.size seems to be expanded into ws.maxMessageSize, ws[ws/rfc6455].ws[ws/rfc6455].maxMessageSize, ws[ws/draft-7x].ws[ws/draft-7x].maxMessageSize in DefaultAcceptOptionsContext

Do we need to put all three values ? How do we handle such cases ?